README.md

```diff
@@ -255,12 +255,53 @@ convention = "google"
 case-sensitive = true
 lines-after-imports = 2
 
+[tool.ruff.lint.flake8-annotations]
+suppress-dummy-args = true
+
 [tool.ruff.lint.extend-per-file-ignores]
-"tests/*" = ["D100", "D103", "ANN"]
+"*" = [
+    "D1", # Ignore warnings about missing docstrings
+    "ANN401", # Ignore warnings about use of the Any type
+]
```

Comment:
I'm not sure about these changed recommendations:
* I don't think we should be ignoring missing docstrings universally
* I think you should be warned about using Any and only be using it as an escape hatch when it's really needed, so I would prefer Anys to have explicit ignore comments in the code to say, yes, we for sure want Any here.